### DISCUSSION TITLE
Rename the "Destroy" button in the music player

### DISCUSSION THREAD

**Discussion by dmarcoux** [navidrome/navidrome#2653] [Ideas]
Hello,

I started to use Navidrome recently and I don't know if anyone was a bit confused by this `Destroy` button in the music player. I am referring to the `X` icon completely at the right side as you can see in this screenshot below:

![example](https://github.com/navidrome/navidrome/assets/1102934/7ad8c41e-33bf-4c7e-b2be-489d2db25d77)

Before clicking the button, I was pretty sure it would simply remove the current queue and indeed, this is what it does. Maybe the wording could instead be `Drop queue` or `Remove queue`.

What do you think?

**Comment by deluan** [comment]
This would need to be changed in the upstream player project. Unfortunately, that project is [unmaintained](https://github.com/navidrome/navidrome/discussions/2285), so there's nothing much we can do until we replace it.
